api.yaml

```
@@ -3506,6 +3506,34 @@ components:
           allOf:
             - $ref: "#/components/schemas/OptionsSymbol"
           nullable: true
+    OptionBrokerageSymbol:
+      description: Option Brokerage symbol
+      type: object
+      properties:
+        id:
+          $ref: "#/components/schemas/Id"
+        description:
+          type: string
+          example: SPY CALL 7/17 200
+        option_symbol:
+          allOf:
+            - $ref: "#/components/schemas/OptionsSymbol"
+          nullable: true
```

```suggestion
          $ref: "#/components/schemas/OptionsSymbol"
```
In this case option_symbol should be non nullable